libpgmodeler/src/table.cpp

```diff
@@ -1998,16 +2002,18 @@ QString Table::getInitialDataCommands(void)
 			commands.append(createInsertCommand(selected_cols, col_values));
 			col_values.clear();
 		}
+		
+		commands.append("\\.");
 
-		return(commands.join('\n'));
+		return(commands.join('\n') + Attributes::DdlEndToken );
 	}
 
 	return(QString());
 }
 
 QString Table::createInsertCommand(const QStringList &col_names, const QStringList &values)
```

Comment:
Maybe this method here can be parameterized in such way to accept a boolean param in which the user (dev) decides if COPY or INSERT need to be generated?

Reply:
I haven't digged UI code good enough to answer and to find out how interaction between UI and codegenerator occurs. 
Probably we should change signature of this method from `QString Table::getInitialDataCommands(void)` to `QString Table::getInitialDataCommands( bool use_copy_syntax = false )` to provide compatibility for models that relied on INSERT syntax.